frontend/src/interfaces/ExperimentCollection.interface.ts

```diff
@@ -0,0 +1,13 @@
+import Experiment from "./Experiment.interface";
+
+export default interface ExperimentCollection {
+    id: number;
+    name: string;
+    description: string;
+    dashboard: Experiment[];
```

Comment:
this can also be null, right?

Reply:
I don't think it can or should. If there's no dashboard it should be an empty array.